setup.py

```diff
@@ -177,6 +179,14 @@ def compilerToPackages():
 
 class InstallCmd(install):
     def run(self):
+        # prepare icons directories
+        os.makedirs('desktop/icons/scalable')
```

Comment:
This doesn't check if the directory already exists. Move the makedirs/copyfile to the end, just before install.run is called.

Reply:
Quote from `help(os.makedirs)`:
```
... any intermediate path segment (not
    just the rightmost) will be created if it does not exist.
```
Yes, I'll move this chunk to the end of `run()` method.